### DISCUSSION TITLE
feat: support for cognee memory

### DISCUSSION THREAD

**Discussion by Vasilije1990** [open-webui/open-webui#6765] [Ideas]
# Feature request

Description:

I would like to propose the integration of Cognee, an open-source memory engine for graphs and vector stores, into the system. Cognee offers a modular framework for handling complex data structures, enabling advanced data retrieval and analysis through knowledge graphs and language models.

GitHub Repository: https://github.com/topoteretes/cognee
Documentation: Available in the repository with examples and detailed guides.

**Comment by sir3mat** [comment]
Are you working on this?
